app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,56 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.users.User;
+import com.codesoom.assignment.domain.users.UserRepository;
+import com.codesoom.assignment.domain.users.UserSaveRequest;
+import com.codesoom.assignment.domain.users.UserUpdateRequest;
+import com.codesoom.assignment.dto.UserUpdateDto;
+import com.codesoom.assignment.exception.UserNotFoundException;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+/**
+ * 회원 조회, 변경 담당
+ */
+@Service
+@AllArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    /**
+     * 회원 저장 후 리턴합니다.
+     *
+     * @param saveRequest 회원 등록에 필요한 데이터
+     */
+    public User save(final UserSaveRequest saveRequest) {
+
+        final User user = User.builder()
+                .email(saveRequest.getEmail())
+                .name(saveRequest.getName())
+                .password(saveRequest.getPassword())
+                .build();
+
+        return userRepository.save(user);
```

Comment:
`userRepository`의 `save` 메소드를 오버로딩해서 `UserSaveRequest`를 받을 수 있도록 작업해보세요. 그러면 이렇게 할 수 있을 겁니다.

```suggestion
        return userRepository.save(saveRequest);
```

Reply:
`save` 메소드 오버로딩해서 구현해 봤습니다. 
회원 저장할 때 서비스에서 `User` 객체를 생성해서 `save` 메소드를 호출하는 게 아닌
`UserSaveRequest` 인터페이스를 그대로 넘김으로써 서비스 레이어 코드가 깔끔해지고
자연스럽다고 느꼈습니다.

Reply:
언제나 더 괜찮은 방법은 있기 마련이죠. 하지만 이 정도로도 충분해 보입니다.

Reply:
넵 계속 더 괜찮은 방법이 있는지 고민해 보겠습니다!